e3smplot/e3sm_utils.py

```diff
@@ -333,6 +333,23 @@ def get_data(dataset, field):
     elif field == 'TMQ':
         if 'tcwv' in dataset.variables.keys():
             data = get_data(dataset, 'tcwv')
+    elif field == 'PS':
+        if 'sp' in dataset.variables.keys():
+            data = get_data(dataset, 'sp')
+    elif field == 'WINDSPD_10M':
+        if 'u10' in dataset.variables.keys() and 'v10' in dataset.variables.keys():
+            10m_zonalwind = dataset['u10']
```

Comment:
I'd prefer if you used `get_data` here because theoretically it should give better error reporting if it can't find the variable in dataset

Reply:
Got it. Just added a commit with that change.